js/super_hero.js

```diff
@@ -1 +1,27 @@
 // var SuperHero = ...
+var SuperHero = function (superName, secretIdentity, superPower) {
+  this.superName = superName;
+  this.secretIdentity = secretIdentity;
+  this.superPower = superPower;
+  this.partner = "";
+};
+
+SuperHero.prototype.useSuperPower = function() {
+  console.log(this.superName + " uses super power " + this.superPower);
+  return this.superName + " uses super power " + this.superPower;
```

Comment:
How could you avoid having to duplicate this logic? Ditto with the method below.